app/utils/sagaInjectors.js

```diff
@@ -29,6 +29,15 @@ const checkDescriptor = (descriptor) => {
   );
 };
 
+const checkRunningSaga = (store, key) => {
+  const descriptor = store.injectedSagas[key];
+  const isRunning = descriptor && descriptor.task && descriptor.task.isRunning();
+  invariant(
+    !isRunning,
```

Comment:
If developers accidentally introduced the same key saga So the original saga will be covered, I think we should not be allowed to pass the same key saga

Reply:
If the saga has already cancelled, the developers should be allowed to pass the same key to start the saga again. Therefore, `checkRunningSaga` should check only if the saga is duplicate and also running.